server/src/graphql/resolvers/driver.ts

```diff
@@ -38,17 +42,20 @@ export default {
     async createDriver(parent: any, args: createDriverArgs, context: any, info: any) {
       return await Driver.signup(args);
     },
-    async loginDriver(_: any, payload:LoginPayload, context) {
+    async loginDriver(parent: any, payload:LoginPayload, context) {
       return await Driver.login(context, payload);
     },
-    async driverCall(root, args : DriverCallArgs, context) {
-      pubsub.publish("driverListen", { driverListen: args });
-      return await args;
+    async driverCall(parent, args : DriverCallArgs, context) {
+      context.pubsub.publish('driverListen', { driverListen: args });
+      return args;
+    },
+    async sendResponse(parent:any, args:DriverResponse, context:any) {
+      return await Trip.sendDriverResponse(args, context);
```

Comment:
```suggestion
      const data = await Trip.sendDriverResponse(args, context);
      // status를 보고 publish를 어떻게 할지에 관한 로직
      // data 중에 필요한 부분만 return
```

Trip service에서 publish 하는 로직이 부자연스러운 것 같아서 status 관련된 부분은 Trip service에서 같이 리턴해서 여기서 validate를 하고 publish를 하는게 어떨까요?

Reply:
수정했습니다!

Reply:
감사합니다.